### PR TITLE
Add blog post on creating Standard compliant repos

### PR DESCRIPTION
**Pull request by Ainali**
Solves #126 
-----
[View rendered _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md](https://github.com/publiccodenet/blog/blob/ja-standard-repos/_posts/2020-12-09-how-to-start-a-standard-compliant-repository.md)

### CODE REVIEW

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+Disclaimer: we are intentionally ignoring the requirement "The codebase SHOULD be in use by multiple parties." in the criteria [Create reusable and portable code](https://standard.publiccode.net/criteria/reusable-and-portable-codebases.html) since no one is using it from the start.
+
+## Step 0: Create an organization
+
+The repository should not be hosted by a single individual. And if you want to comply with spirit of the Standard for Public Code, you should start with creating an organization for this specific codebase. That makes it more attractive for people outside your agency to join in and feel ownership of the codebase. Name the organization the same name as the codebase (org_name below). The name of the project or codebase SHOULD be descriptive and free from acronyms, abbreviations, puns or branding (codebase_name below).
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:21]:
```suggestion
The repository should not be hosted by a single individual. If you want to comply with spirit of the Standard for Public Code, start by creating an organization for this specific codebase. That makes it more attractive for people outside your agency to join in and feel ownership of the codebase. Name the organization the same name as the codebase (org_name below). The name of the project or codebase SHOULD be descriptive and free from acronyms, abbreviations, puns or branding (codebase_name below) - this helps others find it.
```

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+1. Select the organization as the owner.
+2. As `Repository name` write the name of the codebase.
+3. Write a short one sentence description of the codebase.
+4. Set the repository to `Public`.
+5. Don't initialise or choose a license, just create the repository.
+
+## Step 2 - Protect branches
+
+1. Create the codebase directory locally and change directory to it: `mkdir codebase_name`; `cd codebase_name`
+2. Create you repository locally: `git init`
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:36]:
```suggestion
2. Create your repository locally: `git init`
```

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+author: Jan Ainali
+type: blogpost
+excerpt: A step by step guide to make things right right from the start
+categories:
+  - News
+---
+
+# How to start a Standard compliant GitHub repository
+
+As maintainers of the [Standard for Public Code](https://standard.publiccode.net), naturally we want people to be able to follow it as easily as possible. Most commonly we meet people that already have some code in a repository and need to make a few adjustments to meet all of the requirements. But we also meet some people that are about to develop a new codebase, have heard about the Standard and want to meet it from day 1. Which got us thinking about a few practicalities. How do you go about to setup a brand new repository and be standard compliant? In this blog post we will describe how you get started in GitHub. We might return to guides for other platforms in the future.
```
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:13]: I feel like the introduction doesn't accurately signal how much prior knowledge is required - "how to get started in Github" doesn't scream 'you'll need to have a local copy installed and use the command line' to me. 
Is a secondary goal to get people more comfortable with the command line?

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+5. Don't initialise or choose a license, just create the repository.
+
+## Step 2 - Protect branches
+
+1. Create the codebase directory locally and change directory to it: `mkdir codebase_name`; `cd codebase_name`
+2. Create you repository locally: `git init`
+3. Make an initial empty commit: `git commit --allow-empty -m"init"` This is needed to enable branch protection in GitHub.
+4. Add remote: `git remote add origin https://github.com/org_name/codebase_name.git`
+5. Push to GitHub: `git push -u origin master`
+7. Under Settings -> Branches add a new branch protection rule and set it to Require pull request reviews before merging.
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:40]:
```suggestion
7. Under 'Settings' -> 'Branches' add a new branch protection rule and set it to 'Require pull request reviews before merging'.
```

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+* Create a CONTRIBUTING file in the root catalog. In it add:
+    * what writing style guide this repository uses
+    * an email address for security issues and responsible disclosure
+    * guidelines explaining how contributors can get involved
+        * add a link to the issue tracker and pull requests or make clear that you are using GitHub's functionality for these
+        * make clear that issues are an acceptable way for communicating with the maintainers (you may want to change this later to mailing lists or forum)
+* Create a LICENSE file in the root catalog with an [OSI approved license](https://opensource.org/licenses/category).
+* Make sure all text adheres to the writing style guide.
+
+Now you can make a pull request of these changes to the repository. Add a link to the Standard to explain that this repository aims to be standard compliant. If you want to aim higher, do step 4 and 5 before committing. Have someone, preferably from a different context than you, review the pull request before merging.
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:60]:
```suggestion
Now you can make a pull request of these changes to the repository. Add a link to the Standard to explain that this repository aims to be standard compliant. If you want to aim higher, do steps 4 and 5 before committing. Have someone, preferably from a different context than you, review the pull request before merging.
```

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+## Step 4 - Compliance with all SHOULD requirements
+
+* In the README:
+    * add a publicly available roadmap, including information on how it may be influenced
+    * advertise the committed engagement of involved organizations in the development and maintenance
+    * add documentation on the codebase objectives understandable by policy makers and management
+    * add a section for standards used and mention the Standard for Public Code with a link
+* In the CONTRIBUTING:
+    * expand the instructions for how to report potentially security sensitive issues on a closed channel
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:70]:
```suggestion
    * expand the instructions for how to report potentially sensitive security issues on a closed channel
```

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+* Create a CODE OF CONDUCT in the root catalog. If you need help writing it, use [this template](https://www.contributor-covenant.org/version/2/0/code_of_conduct/).
+* In the README:
+    * add documentation on the codebase objectives for the general public
+* Add testing for broken links.
+* In the CONTRIBUTING:
+    * include sections in your style guide on understandable English
+
+## Step 6 - Celebrate
+
+Congratulations, you have set up your repository in a way that is meeting all relevant requirements in the Standard for Public Code. Keep haveing the standard nearby when you start coding as more requirements will become relevant as source code is added to the repository.
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:89]:
```suggestion
Congratulations, your new repository now meets all relevant requirements in the Standard for Public Code. Keep the standard close by when you start coding, as more requirements will become relevant as source code is added to the repository.
```

**Review by ElenaFdR — CHANGES_REQUESTED** [review]
What a helpful blogpost! Here are some proofreading suggestions, plus one big question about the audience.

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Ainali) [reply _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md]: Hmm, no it isn't really a secondary goal, but this way is the one that is most compliant with the Standard. If one only use GitHub, there are some extra commits needed (as far as I know) in order to turn on protected branching.

[the remark this answers is not in the document]
Comment (ElenaFdR) [reply _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md]: I've updated my intro suggestion to more clearly signpost that you need to have a local version available. What do you think?

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+author: Jan Ainali
+type: blogpost
+excerpt: A step by step guide to make things right, right from the start
+categories:
+  - News
+---
+
+# How to start a Standard compliant GitHub repository
+
+As maintainers of the [Standard for Public Code](https://standard.publiccode.net), naturally we want people to be able to follow it as easily as possible. Most commonly we meet people that already have some code in a repository and need to make a few adjustments to meet all of the requirements. But we also meet some people that are about to develop a new codebase, have heard about the Standard and want to meet it from day 1...which got us thinking about a few practicalities. How do you go about to setting up a brand new repository and being Standard compliant?
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:13]:
```suggestion
As maintainers of the [Standard for Public Code](https://standard.publiccode.net), naturally we want people to be able to follow it as easily as possible. Most commonly we meet people that already have some code in a repository and need to make a few adjustments to meet all of the requirements. But we also meet some people that are about to develop a new codebase, have heard about the Standard and want to meet it from day 1...which got us thinking about a few practicalities. How do you go about setting up a brand new repository and being Standard compliant?
```

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+In this blog post we describe how you get started in GitHub (assuming you have a basic familiarity with GitHub). We may return to guides for other platforms in the future.
```
Comment (ElenaFdR) [inline _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md:15]:
```suggestion
In this blog post we describe how you get started using GitHub (assuming you're familiar with Github and can install a local version). We may return to guides for other platforms in the future.
```

File: _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ElenaFdR) [reply _posts/2020-12-09-how-to-start-a-standard-compliant-repository.md]: I've made a new suggestion below!

**Review by ElenaFdR — APPROVED** [review]
Almost ready to whoosh off into the world!

**Pull request merged**